app/models.py

```diff
@@ -1551,17 +1558,26 @@ def serialize(self):
     AuditEvent.created_at,
 )
 
-# Index for type + acknowledged audit event query count - for admin
-# app "Service updates" requests: while the page of audit events
-# will rely on the created_at index scan and is relatively quick,
-# count(*) query executed by `.paginate` is very slow without a
-# dedicated index for objects with multiple events
+
+# DEPRECATED - remove in a migration once supplier update admin app has been updated
 db.Index(
     'idx_audit_events_type_acknowledged',
     AuditEvent.type,
     AuditEvent.acknowledged,
 )
 
+# partial index for supplier_update-monitoring admin views. specifically this
```

Comment:
Pas du tout. If the feature gets expanded to another object type, we just add the new update event to the index condition. No new code needs to be written, no new tests. It's a one-liner (apart from the migration...).

And it's not like we don't have existing precedents for adding e.g. specific indexes targeted at specific things "we know" the frontend(s) do. The replaced comment here for the previous index was something about "this is for the supplier updates admin page" and it was set up to use a _specific set_ of filters that that particular frontend view used.